setup.py

```diff
@@ -2,13 +2,40 @@
 
 setup(
     name='osmviz',
-    version='1.0',
+    version='1.0.1',
+    description=
+        'OSMViz is a small set of Python tools for retrieving '
+        'and using Mapnik tiles from a Slippy Map server '
+        '(you may know these as OpenStreetMap images).',
     long_description=open('README.md', 'r').read(),
-    packages= find_packages('src'),
-    package_dir = {'': 'src'},
-    zip_safe=False,
+    classifiers=[
+        #'Development Status :: 5 - Production/Stable',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.5',
+        'Programming Language :: Python :: 2.6',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.1',
+        'Programming Language :: Python :: 3.2',
+        'Programming Language :: Python :: 3.3',
```

Comment:
Let's ditch Python 2.5, 2.6, 3.1 and 3.2, they're no unsupported even as CPython.

https://en.wikipedia.org/wiki/CPython#Version_history